frontend/src/metadata/popups/SliceFinderPopup.svelte

```diff
@@ -133,7 +153,25 @@
 		</div>
 		<div class="inline">
 			<div style:margin-left={"20px"}>
-				<div class="options-header">Metric Column</div>
+				<div style="display:flex">
+					<div class="options-header">Metric Column</div>
+					<div
+						class="information-tooltip"
+						style="margin-top: 3px;"
+						use:tooltip={{
+							content:
+								$tab !== "comparison"
+									? "The metric column to decide performance"
+									: "The metric column that has difference between models",
```

Comment:
"The continuous column to compare slices across". Can be the same for both I think